src/favorites/api.js

```diff
@@ -1,20 +1,14 @@
-import { getByPath, setByPath } from '../firebase/initialize';
+import { getByPath, setByPath } from '../firebase';
 import { Category } from './models';
 
-export const updateFavorite = (userId, favoriteId, isFavorite, category) => (
-  setByPath(`favorites/${userId}/${category}/${favoriteId}`, isFavorite)
-);
+export const updateFavorite = (userId, favoriteId, isFavorite, category) =>
+  setByPath(`favorites/${userId}/${category}/${favoriteId}`, isFavorite);
 
-export const fetchFavorites = (userId) => (
-  getByPath(`favorites/${userId}`)
-    .then(toFavorites)
-);
+export const fetchFavorites = userId => getByPath(`favorites/${userId}`).then(toFavorites);
```

Comment:
We might want to consider setting up an auto-formatting configuration like prettier so that we don't have this kind of stuff

Reply:
Yeah, I use prettier and having stricter linting rules will help as well. Some of these changes seem random in context but I hope when the style rules are better defined it makes more sense

Reply:
I don't really care what our lint rules are.  I have been mentally using my memory of airbnb, but if you want to just check in your prettier config and run it on commit, that seems useful

Reply:
though maybe do it in a different pr, since this one is already so big

Reply:
Yeah the commits that I removed in order to trim the size of this PR included extending the airbnb config